### DISCUSSION TITLE
Simple script to send result to DefectDojo

### DISCUSSION THREAD

**Discussion by SPoint42** [security-code-scan/security-code-scan#192] [Show and tell]
If you  want to use https://github.com/security-code-scan/security-code-scan and analyze the result to send to Defectdojo,  I wrote a simple script to analyse the output and send it to DefectDojo : https://github.com/SPoint42/SecurityTools/tree/main/SCDotNet2DefectDojo

Next pull will be the integration of the output of CWE from the tools ; see => https://github.com/security-code-scan/security-code-scan/pull/185

**Comment by JarLob** [comment]
I have added your script to documentation and release notes. Thanks again for the contribution!

**Reply by SPoint42** [reply]
I'm in the process to write the Defectdojo Parser. See https://github.com/DefectDojo/django-DefectDojo/pull/4479
